packages/cactus-plugin-keychain-aws-sm/src/test/typescript/integration/a.ts

```diff
@@ -0,0 +1,15 @@
+// import fs from "fs"
```

Comment:
@jagpreetsinghsasan If this is just a leftover scratchpad then it could be deleted.